exchange/bidder.go

```diff
@@ -132,7 +132,8 @@ type bidderAdapterConfig struct {
 }
 
 func (bidder *bidderAdapter) requestBid(ctx context.Context, bidderRequest BidderRequest, conversions currency.Conversions, reqInfo *adapters.ExtraRequestInfo, adsCertSigner adscert.Signer, bidRequestOptions bidRequestOptions, alternateBidderCodes openrtb_ext.ExtAlternateBidderCodes, hookExecutor hookexecution.StageExecutor, ruleToAdjustments openrtb_ext.AdjustmentsByDealID) ([]*entities.PbsOrtbSeatBid, extraBidderRespInfo, []error) {
-	reject := hookExecutor.ExecuteBidderRequestStage(bidderRequest.BidRequest, string(bidderRequest.BidderName))
+	brw := openrtb_ext.RequestWrapper{BidRequest: bidderRequest.BidRequest}
+	reject := hookExecutor.ExecuteBidderRequestStage(&brw, string(bidderRequest.BidderName))
```

Comment:
Seems acceptable to me until we can pull the RequestWrapper down to this level. I believe the goal with this PR is to unify the bidder request payload across all module hooks, preferring the RequestWrapper. Right?

We need to be sure that after applying the change set, the request wrapper is synced back with the request. Is this accounted for?

Reply:
Yes, good point, I was not sure about this. Technically we pass request by pointer inside of request wrapper and request modifications inside of modules are done through the mutations functions. It works properly with what we have now, for instance existing mutations like update BAdv, CatTax etc. But you are right, I think we need to rebuild request and set it back to to `bidderRequest` because modules have access to actual payload and can modify request using request wrapper.